src/main/java/org/gridsuite/filter/server/dto/FormFilter.java

```diff
@@ -38,7 +38,7 @@ public FormFilter(UUID id, Date creationDate, Date modificationDate, AbstractEqu
     @JsonProperty(access = JsonProperty.Access.READ_ONLY)
     @Override
     public FilterType getType() {
-        return FilterType.FORM;
+        return FilterType.AUTOMATIC;
     }
 
     public AbstractEquipmentFilterForm getEquipmentFilterForm() {
```

Comment:
I see it's the same here ... Do we need this getter implementation ?

Reply:
We don't have a type attribute in this class, we can't do it with the @Getter